ui/src/js/viewModels/orgAdmin.js

```diff
@@ -7,436 +7,453 @@
+                        "user_approver": self.userAdmin().length > 0 ? "Y" : "N",
+                        "need_approver": self.userAdmin().length > 0 ? "Y" : "N",
+                        "manage_offers": "Y",
+                        "manage_clients": "Y",
+                        "client_share_approver": self.userAdmin().length > 0 ? "Y" : "N",
+                        "dbs_check": "P",
+                        "confirmed": "Y"
 
-                                },
-                                error = function (response) {
-                                    self.postText("Error: User changes not saved.");
-                                    self.postTextColor("red");
-                                    console.log("user data not posted");
-                                }).then(function () {
+                    };
+
+                    return $.when(restClient.doPostJson('/rest/user_organizations', userData)
+                        .then(
+                            success = function (response) {
+                                self.postText("You have succesfully saved added the user to the organization.");
+                                self.postTextColor("green");
+                                document.getElementById('duplicateUserDialog').close();
+                                document.getElementById('addUserDialog').close();
+                                self.getUserOrgData(self.userOrgId());
+
+                            },
+                            error = function (response) {
+                                self.postText("Error: User changes not saved.");
+                                self.postTextColor("red");
+                                console.log("user data not posted");
+                            }).then(function () {
                                 self.fileContentPosted(true);
                                 $("#postMessage").css('display', 'inline-block').fadeOut(2000, function () {
                                     //self.disableSaveButton(false);
                                 });
                             })
-                        );
-					  }
+                    );
+                }
 
-					self.saveUserButton = function () {
+                self.saveUserButton = function () {
 
-                        var userData =
-                            {
-                                "organization_id":self.orgDetailid(),
-                                "user_name": self.userName(),
-                                "email": self.userEmail(),
-								"phone": null,
-								"admin":"N"
-                            };
+                    var userData =
+                    {
+                        "organization_id": self.userOrgId(),
+                        "user_name": self.userName(),
+                        "email": self.userEmail(),
+                        "phone": null,
+                        "admin": "Y"
```

Comment:
I think I accidentally left in a change from my testing. However, I'm not sure if you saw what I said about there being two saveUserButton functions, but this is part of the first of these functions. This saveUserFunction that starts on line 198 does not seem to have functionality since I tried commenting it out and as far I could tell nothing changed. Do you know why there are two saveUserButton functions, or shall we remove the one that isn't being used.

Reply:
lYes, there is sometimes old code in there. If you are happy it is no longer required, delete it. I will likely ask in the review comments why, but please do realize that a question of why isn't the same as saying "don't do it".

Reply:
I have deleted the unused function - I have fully tested the org admin tab and everything still works (besides from a couple of bugs we already knew about).